systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicScalabilityUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.kafkaUtils;
+
+import io.strimzi.api.kafka.model.KafkaTopicSpec;
+import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
+import io.strimzi.systemtest.templates.crd.KafkaTopicTemplates;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import io.strimzi.systemtest.resources.ResourceManager;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import io.strimzi.systemtest.enums.CustomResourceStatus;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+
+public class KafkaTopicScalabilityUtils {
+    private static final Logger LOGGER = LogManager.getLogger(KafkaTopicUtils.class);
+
```

Comment:
There should be a private constructor to not allow users to create instances of such class. (see other util classes)